readthedocs/proxito/views/hosting.py

```diff
@@ -514,6 +512,25 @@ def _v1(self, project, version, build, filename, url, request):
             },
         }
 
+        # Show the subprojects filter on the parent project and subproject
+        # TODO: Remove these queries and try to find a way to get this data
+        # from the resolver, which has already done these queries.
+        if project.subprojects.exists():
+            data["addons"]["search"]["filters"].append(
+                [
+                    "Include subprojects",
+                    f"subprojects:{project.slug}/{version.slug}",
+                ]
+            )
+        if project.superprojects.exists():
+            superproject = project.superprojects.first().parent
+            data["addons"]["search"]["filters"].append(
+                [
+                    "Include subprojects",
```

Comment:
If you are reading a subproject, you want to get results from sibling projects -- since subprojects don't have other subprojects, but sibling ones.

```suggestion
                    "Include sibling projects",
```